frontend/src/redux/organiser/actionTypes.js

```diff
@@ -9,9 +9,12 @@ export const UPDATE_GAVEL_ANNOTATORS = 'organiser/UPDATE_GAVEL_ANNOTATORS'
 export const UPDATE_RANKINGS = 'organiser/UPDATE_RANKINGS'
 
 export const REMOVE_ORGANISER = 'organiser/REMOVE_ORGANISER'
-
 export const ADD_ORGANISER = 'organiser/ADD_ORGANISER'
 
+export const UPDATE_EVENT_RERUITERS = 'organiser/UPDATE_EVENT_RERUITERS'
```

Comment:
- [ ] Required: There's a typo on 'Recruiter', it would be better to fix it to avid confusion